workspaces/orchestrator/scripts/postinstall

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+mydir=`pwd`
+cd ../../ && yarn install
+cd $mydir
+cd plugins/orchestrator-common
+yarn openapi:check
```

Comment:
I thought the idea was to remove the whole generated code and run yarn openapi:generate
Not check it here

Reply:
ok